packages/vite/src/node/plugins/css.ts

```diff
@@ -2273,7 +2273,9 @@ async function compileLightningCSS(
           },
         },
         minify: config.isProduction && !!config.build.cssMinify,
-        sourceMap: config.css?.devSourcemap,
+        sourceMap: config.isProduction
+          ? !!config.build.sourcemap
+          : config.css?.devSourcemap,
```

Comment:
`config.isProduction` doesn't mean it's in `vite build` (e.g. `NODE_ENV=development vite build`). So I think using `config.command === 'build'` is correct here.
But given that CSS sourcemaps in build is not supported yet (#2830), does this fix anything? (I think we can still have it but it's `chore:` than a `fix:` in that case)

Reply:
This fixes annoying warning 'vite:css didn't provide source map' when using source map in production but not `devSourcemap`

Reply:
Ah, I see. 👍